docs/release-notes/release-notes-1.7.0.md

```diff
@@ -0,0 +1,1434 @@
+* Support for a configurable assumed valid block
+* Block index memory usage reduction
+* Asynchronous indexing
+* Version 1 block filters removal
+* Various updates to the RPC server:
+  * Additional per-connection read limits
+  * A more strict cross origin request policy
+  * A new alternative client authentication mechanism based on TLS certificates
+  * Availability of the scripting language version for transaction outputs
+  * Various other RPC modifications, additions, and removals
+* New developer modules:
+  * Age-Partitioned Bloom Filters
+  * Fixed-Precision Unsigned 256-bit Integers
+  * Standard Scripts
+  * Standard Addresses
+* Infrastructure improvements
+* Quality assurance changes
+
+For those unfamiliar with the
+[voting process](https://docs.decred.org/governance/consensus-rule-voting/overview/)
+in Decred, all code needed in order to support each of the aforementioned
+consensus changes is already included in this release, however it will remain
+dormant until the stakeholders vote to activate it.
+
+For reference, the consensus change work for each of the four changes was
+originally proposed and approved for initial implementation via the following
+Politeia proposals:
+- [Decentralized Treasury Spending](https://proposals-archive.decred.org/proposals/c96290a)
+- [Explicit Version Upgrades Consensus Change](https://proposals.decred.org/record/3a98861)
+- [Automatic Ticket Revocations Consensus Change](https://proposals.decred.org/record/e2d7b7d)
+- [Change PoW/PoS Subsidy Split From 60/30 to 10/80](https://proposals.decred.org/record/427e1d4)
+
+The following Decred Change Proposals (DCPs) describe the proposed changes in
+detail and provide full technical specifications:
+- [DCP0007](https://github.com/decred/dcps/blob/master/dcp-0007/dcp-0007.mediawiki)
+- [DCP0008](https://github.com/decred/dcps/blob/master/dcp-0008/dcp-0008.mediawiki)
+- [DCP0009](https://github.com/decred/dcps/blob/master/dcp-0009/dcp-0009.mediawiki)
+- [DCP0010](https://github.com/decred/dcps/blob/master/dcp-0010/dcp-0010.mediawiki)
+
+**It is important for everyone to upgrade their software to this latest release
+even if you don't intend to vote in favor of the agenda.**
+
+## Downgrade Warning
+
+The database format in v1.7.0 is not compatible with previous versions of the
+software.  This only affects downgrades as users upgrading from previous
+versions will see a one time database migration.
+
+Once this migration has been completed, it will no longer be possible to
+downgrade to a previous version of the software without having to delete the
+database and redownload the chain.
+
+The database migration typically takes around 40-50 minutes on HDDs and 20-30
+minutes on SSDs.
+
+## Notable Changes
+
+### Four New Consensus Change Votes
+
+Four new consensus change votes are now available as of this release.  After
+upgrading, stakeholders may set their preferences through their wallet.
+
+### Revert Treasury Maximum Expenditure Policy Vote
```

Comment:
Suggest making all four of these headings at fourth level (`####`) so they are nested under the heading `### Four New Consensus Change Votes`

Reply:
Changed.  I went back and forth on this and had decided to leave it with 3 so they stand out better as 4 makes it a bit small, but I agree that they really should be sub headings.